hws/README.md

```diff
@@ -47,6 +47,7 @@ Running syntax
 | -pid DPDK_PID | Trigger DPDK app <PID> |
 | -port DPDK_PORT | Trigger DPDK app <PORT> (must provide PID with -pid) |
 | -extra_hw_res [pat, arg] | Request extra HW resources to be dumped. For example: -extra_hw_res pat,arg |
+| -s | Show dump statistics. |
```

Comment:
"Show dump statistics" - what does it mean? Maybe "Dump steering statistics"?
Also, some additional info on what is dumped is welcome.
E.g.: "Dump steering statistics, such as bla and bla"

Reply:
i think show dump statistics more suitable as we show stats about the dump itself. but added more info